llvm/lib/YkIR/YkIRWriter.cpp

```diff
@@ -465,6 +458,19 @@ class YkIRWriter {
     for (unsigned OI = 0; OI < I->arg_size(); OI++) {
       serialiseOperand(I, VLMap, I->getOperand(OI));
     }
+    if (!I->getCalledFunction()->isDeclaration()) {
+      // The next instruction will be the stackmap entry
+      CallInst *SMI = dyn_cast<CallInst>(I->getNextNonDebugInstruction());
+      assert(SMI);
+      assert(SMI->getCalledFunction()->isIntrinsic());
+      assert(SMI->getIntrinsicID() == Intrinsic::experimental_stackmap);
+      // has_safepoint = 1:
```

Comment:
I wonder if we should rename this back to "has_stackmap" now. Because we are attaching an LLVM stackmap to a call/condbr. We use the stackmap ID to lookup the stackmap in the binary and parse it using a stackmap parser. I personally find it confusing that this is suddenly being called "safepoint".

Reply:
This is resolved now. See discussion in https://github.com/ykjit/yk/pull/1126